web_console/middlewares/find_options.js

```diff
@@ -0,0 +1,19 @@
+// Common middleware for Sequelize FindOptions generation
+
+module.exports = async function FindOptionsMiddleware(ctx, next) {
+  ctx.findOptions = {};
+
+  if (ctx.headers['x-federation-id']) {
+    const federation_id = parseInt(ctx.headers['x-federation-id'], 10);
+    if (Number.isNaN(federation_id) || federation_id < 1) {
+      ctx.status = 400;
+      ctx.body = {
+        error: 'Invalid header: X-Federation-Id',
+      };
+      return;
+    }
+    ctx.findOptions.where = { federation_id };
```

Comment:
is `{[Op.eq]: federation_id}` needed here?

Reply:
they are equivalent here